lang/po/new/cs.po

```diff
@@ -1533,7 +1529,7 @@ msgstr "Kalibrace XYZ v poradku. X/Y osy mirne zkosene. Dobra prace!"
 # MSG_TIMEOUT c=12
 #: messages.c:157
 msgid "Timeout"
-msgstr ""
+msgstr "Timeout"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.